Modification de l'affichage lors de la validation du mot de passe

2265857 opened the pull request:
Affichage de la validation du mot de passe uniquement lorsque l'utilisateur clic sur le bouton Créer.

Linkky committed:
Code fonctionnel

PlamediIlunga reviewed (commented):
Présence d'un span qui donne des indications à l'utilisateur des critères du mot de passe . À vérifier si c'est le bon endroit ou le placer .Lignes (226-228)
Quelques changements dans le css (global.css , register.css) .À vérifier si demandé ou pas

VincentBouchar-d reviewed (dismissed):
C'est beau ! Peut-être dire dans le premier commentaire de la PR les classes qui sont utilisées pour le CSS, ça va faciliter la lecture des changements.

antoineCegepRDL commented on front-end/package-lock.json:
```diff
@@ -0,0 +1,6 @@
+{
```
Fichier inutile, (à retirer)

2265857 replied:
Fichier supprimé.

antoineCegepRDL commented on front-end/projet_application/src/routes/register/+page.svelte:
```diff
@@ -94,11 +94,17 @@
     }
 
+    // État pour suivre si le formulaire a été soumis afin d'afficher la validation des critères de mot de passe
+    let formSubmitted = false
```
retirer le commentaire et donne un nom plus significatif comme `showPasswordValidations`

2265857 replied:
Changement du nom de la variable formSubmitted ➠ showPasswordValidations

Linkky committed:
Correctif fait selon les commentaires du PullRequest fait par les pairs:
• Suppression du fichier package-lock.json du front-end
• Changement nom variable pour nom plus significatif ➠ showPasswordValidations

VincentBouchar-d reviewed (approved):
LGTM
